### DISCUSSION TITLE
LoanCore and HDMA individuals

### DISCUSSION THREAD

**Discussion by VladimirAlexiev** [edmcouncil/fibo#1371] [Loans (LOAN) & Market Data (MD)]
https://spec.edmcouncil.org/fibo/ontology/LOAN/LoanContracts/LoanCore/ and
https://spec.edmcouncil.org/fibo/ontology/LOAN/LoanContracts/LoanHMDA/ have a number of individuals that violate naming convention, eg

```
fibo-loan-ln-ln:_ConstructionType_onFrameModular
fibo-loan-ln-hmda:_HMDA_Disposition_PreapprovalRequestDenied
fibo-loan-ln-ln:_PublicRecordCategory_lien
fibo-loan-ln-ln:_ManufacturedHomeLegalClassification_realProperty
```

Also, some but not all of them are declared `owl:Thing` (in addition to `owl:NamedIndividual`).

Maybe it's better making namespaces for these taxonomies and renaming to eg
```
fibo-loan-ln-ln-constr:onFrameModular
fibo-loan-ln-hmda-disp:PreapprovalRequestDenied
fibo-loan-ln-ln-rec:lien
fibo-loan-ln-ln-home:realProperty
```

**Comment by ElisaKendall** [comment]
There are lots of issues remaining in these ontologies - not limited to the individuals (they are still provisional).  We did a little work to being to merge them with other FIBO Loan ontologies in Q1, but there is more work to do on them.  This is just one example.
